content/glossary/sql/delete.md

```diff
@@ -0,0 +1,38 @@
+<!--
```

Comment:
remove "sql" from tags and subjects, title from markdown

Reply:
👍🏼